scripts/package

```diff
@@ -23,9 +23,18 @@ DAPPER_BASE_IMAGE=${REPO}/dapper-base:${TAG}
 # We expect the Dapper base image to change rarely, so pull it first
 docker pull ${REPO}/dapper-base:latest || :
 
+CHANGED_FILES_LOCAL=$(git diff --name-only)
```

Comment:
Changing this to `git diff --name-only origin/master` would ensure that the base image is rebuilt for any local change, whether committed or otherwise, assuming there’s an `origin` remote.

Reply:
As I said, that's not the case on my side.  And there are corner cases like.. PRs that go on top of a branch (easy to detect in travis, but hard to detect on  local git, etc).

Saying that, I think we should also add this detection on the deploy phase, so deployment of this image only happens when a change has been really made.

And anyway, before it wasn't better, because before we were building dapper-base:*dev*, which does not lead to running dapper with that version on later steps (since we use dapper-base:later), I could be missing something

Reply:
Resolved on different PR